public/Lab-Files/Lesson03/Challenges/apple-bananas.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <style>
+    .apples {color: red;}    
+    .bananas {color: yellow;}   
```

Comment:
Hey Wesley, great job on these challenges. 

During class I always see you working hard and actually understanding  most assignments. That is reflected on your code, awesome job!